### ISSUE TITLE
Cannot find module './git-util'

### ISSUE THREAD

**Issue by uqe** [mathdroid/crop-github-images-cli#3]
I tried both npx/npm i -g

Version: 1.4.0
OS version: macOS 10.14.4
Node version: 11.14.0

```
internal/modules/cjs/loader.js:670
    throw err;
    ^

Error: Cannot find module './git-util'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/usr/local/lib/node_modules/crop-github-images-cli/cli.js:15:44)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
```

**Comment by mathdroid** [comment]
Gotcha. I forgot to include `gif-util` in published files. Will fix this promptly. Thanks @uqe!

**Comment by mathdroid** [comment]
Fixed in [1.4.1](https://github.com/mathdroid/crop-github-images-cli/releases/tag/v1.4.1)
